package.json

```diff
@@ -115,21 +115,22 @@
     "eslint-config-airbnb": "^3.1.0",
     "eslint-plugin-import": "^2.8.0",
     "expose-loader": "^0.7.1",
-    "extract-text-webpack-plugin": "^3.0.0",
     "http-server": "^0.11.2",
     "jsdom": "^11.7.0",
     "lodash": "^4.17.21",
+    "mini-css-extract-plugin": "^0.12.0",
     "mocha": "^9.2.2",
     "mocha-jenkins-reporter": "^0.1.9",
-    "node-sass": "^4.14.1",
     "nyc": "^15.0.0",
     "prettier": "^1.19.1",
     "proxyquire": "1.7.4",
-    "sass-loader": "6.0.7",
+    "sass": "^1.93.2",
```

Comment:
Any reason for a newer version of sass here than in immerse ("^1.77.0"), i'd think it would be best to be consistent across our repos if possible, if not 🤷  but we should if we can

Reply:
Upgraded `sass` and `sass-loader` to match immerse - also needed to upgrade to webpack 5 for `sass-loader`